apis/api-directory/allocation-api.md

```diff
@@ -0,0 +1,59 @@
+# Allocation API
+
+{% swagger method="get" path="/allocation/query" baseUrl="http://app.kubecost.com/query" summary="Allocation API (Querying)" %}
+{% swagger-description %}
+The Allocation API is the preferred way to query for costs and resources allocated to Kubernetes workloads and optionally aggregated by Kubernetes concepts like `namespace`, `controller`, and `label`.
+{% endswagger-description %}
+
+{% swagger-parameter in="path" name="window" type="string" required="true" %}
+Duration of time over which to query. Accepts multiple formats including units of time, relative time units, and unix timestamps. See this section on [using the `window` parameter](/apis/api-directory/api-directory.md#using-the-window-parameter) for more information.
+{% endswagger-parameter %}
+
+{% swagger-parameter in="path" name="format" type="string" required="false" %}
+File type when exporting query. Currently only supports `json`.
+{% endswagger-parameter %}
+
+{% swagger-parameter in="path" name="aggregate" type="string" required="false" %}
+Field by which to aggregate the results. Accepts: `cluster`, `namespace`, `controllerKind`, `controller`, `service`, `node`, `pod`, `label:<name>`, and `annotation:<name>`. Also accepts comma-separated lists for multi-aggregation, like `namespace,label:app`.
+{% endswagger-parameter %}
+
+{% swagger-parameter in="path" name="accumulate" type="boolean" required="false" %}
+If `true`, sum the entire range of time intervals into a single set. Default value is `false`. Also supports accumulation by specific intervals of time including `hour`, `day`, and `week`. Does not accumulate idle costs, which must be configured separately.

```

Comment:
We actually **don't** support accumulation by `hour`, `day` or `week` in KC Cloud. 
```suggestion
If `true`, sum the entire range of time intervals into a single set. Default value is `false`. Does not accumulate idle costs, which must be configured separately.

```

Reply:
Does that only refer to the `accumulate` parameter, or `window` also? Window uses a lot of different relative params in self-hosted like  `yesterday`, `lastweek`, etc. Are those valid? Haven't read all your comments but we should have a specific paragraph on using the `window` param bc of how common it is in our APIs